maui-toolkit-toc.html

```diff
@@ -256,6 +257,7 @@
             <li><a href="/maui-toolkit/NumericUpDown/Formatting">Formatting</a></li>
             <li><a href="/maui-toolkit/NumericUpDown/Restriction">Restriction</a></li>
             <li><a href="/maui-toolkit/NumericUpDown/Updown-button">UpDown Button</a></li>
+            <li><a href="/maui-toolkit/NumericUpDown/Events">UpDown Button</a></li>
```

Comment:
Use the topic name also Events instead of UpDown Button

Reply:
updated